bublik/interfaces/api_v2/config.py

```diff
@@ -188,8 +188,8 @@ def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
```

Comment:
Don't use past tense in the summary and description.

Reply:
Fixed